lib/authentication/service/auth_provider.dart

```diff
@@ -66,6 +66,7 @@ class AuthProvider with ChangeNotifier {
             AppToast.show(S.current.errorInvalidEmail);
             break;
           case 'ERROR_WRONG_PASSWORD':
+          case 'wrong-password':
```

Comment:
Huh? What's this?

Reply:
I attached a photo, at least on web, if you enter your password wrong the error code of the action is 'wrong-password' not 'ERROR_WRONG_PASSWORD'.

Reply:
Interesting, but how about the other errors? Does the name correspond there? Let's not change just one and leave the others broken.